libsolidity/codegen/ABIFunctions.cpp

```diff
@@ -354,8 +375,35 @@ string ABIFunctions::conversionFunction(Type const& _from, Type const& _to)
 			break;
 		}
 		case Type::Category::FixedPoint:
-			solUnimplemented("Fixed point types not implemented.");
+		{
+			FixedPointType const &fromFixedType = dynamic_cast<FixedPointType const &>(_from);
```

Comment:
The `&` is glued to the type, not the name, i.e. it should be `FixedPointType const& fromFixedType`